torchvision/prototype/transforms/functional/__init__.py

```diff
@@ -1,27 +1,54 @@
-from ._augment import erase_image, mixup_image, mixup_one_hot_label, cutmix_image, cutmix_one_hot_label
+from . import utils  # usort: skip
+
+from torchvision.transforms import InterpolationMode
+
+from ._augment import (
+    erase_image,
```

Comment:
Exactly. Due to the dynamic nature of the dispatch as well as the extra meta data on the `Feature`'s, we can't script the dispatchers. The low level kernels enable users to have this functionality at the cost of inconvenience.

Reply:
Personally I would keep them public but wouldn't expose these from `__init__`. We want to push people to use the high-level kernels. Those who want low-level could import directly. Unfortunately that is not possible because all the subpackages are private (for example `_augment`). We can keep as-is and examine potentially exposing these from somewhere else.

Reply:
> We want to push people to use the high-level kernels.

Is that the case? I was under the impression that scriptable kernels is an integral part of some workflows. Why would make it harder for these users?

Reply:
I'll propose something in a follow-up PR.

Reply:
Instead of creating PRs directly, create an issue outlining the problem. This requires discussion as there are alternatives. I just want to avoid biasing towards one direction without discussion.